gamedb.yaml

```diff
@@ -5493,3 +5493,745 @@
+  banner: https://cdn2.steamgriddb.com/file/sgdb-cdn/grid/5f8b73c0d4b1bf60dd7173b660b87c29.png
+  background: https://cdn2.steamgriddb.com/file/sgdb-cdn/hero/7ca862b72603622054d537b558eb9b9c.png
+  logo: https://cdn2.steamgriddb.com/file/sgdb-cdn/logo/3210806875ddb9745d3d8313e65b81ab.png
+  icon: https://cdn2.steamgriddb.com/file/sgdb-cdn/icon/1e8a19426224ca89e83cef47f1e7f53b.ico
+  status: playable
+  notes:
+    - Displays keyboard icons
+
+- name: "Betrayer"
+  platform: gog
+  id: 1090734724
+  banner: steam:243120
+  poster: https://cdn2.steamgriddb.com/grid/9fc497b1718cabc904e5af82c57bae53.png
+  background: https://cdn2.steamgriddb.com/hero/69a4241986721658eb6237d56c1b8abc.jpg
+  logo: https://cdn2.steamgriddb.com/logo/1145a30ff80745b56fb0cecf65305017.png
+  icon: https://cdn2.steamgriddb.com/icon/60c43e386965167ba97045a1fe957626.png
+  compat_tool: proton-stable
+  status: playable
+  notes:
+    - Displays kexboard icons
+
+- name: "Butcher"
+  platform: gog
+  id: 1689871374
+  banner: steam:474210
+  status: playable
+  notes:
+    - Controller not working
+
+- name: "Broken Sword: Director's Cut"
+  platform: gog
+  id: 1207658900
+  banner: steam:57640
+  status: playable
+  notes:
+    - No controller support
+
+- name: "BEAUTIFUL DESOLATION"
+  platform: gog
+  id: 1295420179
+  banner: steam:912570
+  status: playable
+  notes:
+    - No controller support
+
+- name: "Helium Rain"
+  platform: gog
+  id: 1807214820
+  banner: steam:681330
+  status: playable
+  notes:
+    - Tutorial shows keyboard icons
+
+- name: "Hero of the Kingdom II"
+  platform: gog
+  id: 1666688320
+  banner: steam:346560
+  status: playable
+  notes:
+    - No controller support
+
+- name: "Ozymandias: Bronze Age Empire Sim"
+  platform: gog
+  id: 1166262945
+  banner: steam:1768280
+  status: playable
+  notes:
+    - No controller support
+
+- name: "Jazz Jackrabbit 2: The Christmas Chronicles"
+  platform: gog
+  id: 1597842603
+  banner: https://cdn2.steamgriddb.com/grid/5280fc5bd764f3387f774f273c4b3f01.png
+  poster: https://cdn2.steamgriddb.com/grid/c81632061ae1a7602021f5669357cc34.png
+  background: https://cdn2.steamgriddb.com/hero/89eec4cf449e1db057297d6f9304a99e.png
+  logo: https://cdn2.steamgriddb.com/logo/2d5c1f9d0ff0d0bc3f0465f0ff1ff653.png
+  icon: https://cdn2.steamgriddb.com/icon/d869c99656ec60fc9de27338a87b2506.ico
+  status: playable
+  notes:
+    - Limited controller support
+
+- name: "Jazz Jackrabbit 2: The Secret Files"
+  platform: gog
+  id: 1351891846
+  banner: https://cdn2.steamgriddb.com/grid/5391707bb1c266d61d0c6d1e7948faaa.png
+  poster: https://cdn2.steamgriddb.com/grid/d6f993113df8012c1868e8612a279693.png
+  background: https://cdn2.steamgriddb.com/hero/8cff9bf6694dccfc3b6a613d05d51d16.png
+  logo: https://cdn2.steamgriddb.com/logo/54cf07b091f36d3e0a2d1e5bcf65d66a.png
+  icon: https://cdn2.steamgriddb.com/icon/d6d83fd1acd1918a677906afc9bdc1d7.ico
+  status: verified
```

Comment:
Is "verified" correct? The note says "limited controller support".

Reply:
Sorry. Playable is correct.